awx/main/tasks.py

```diff
@@ -1103,7 +1103,7 @@ def run(self, pk, **kwargs):
                                           start_args='')  # blank field to remove encrypted passwords
 
         self.instance.websocket_emit_status("running")
-        status, rc, tb = 'error', None, ''
+        status, rc, = 'error', None
```

Comment:
the comma after `rc` might be allowable, but it's weird

Reply:
Yea, agreed, I'll address it.